profiles/chlcc/achievementsystem.lua

```diff
@@ -0,0 +1,4 @@
+root.AchievementData = {
+  Type = AchievementDataType.CHLCC,
```

Comment:
Same here, the type I think should be "PS3" rather than CHLCC, TROPHY.TRP is a PS3 file format rather than a game format